std_msgs/Header header
int64 volumeRate_cmd        # volumeRate_cmd reading it could be from 0-5

